lib/apipie_bindings/rest_client_extensions.rb

```diff
@@ -0,0 +1,32 @@
+require 'rest_client'
+module ApipieBindings
+  module RestClientExtensions
+
+    module RequestAccessor
+      attr_accessor :request
+    end
+
+    RestClient::AbstractResponse.send(:include, RequestAccessor) unless RestClient::AbstractResponse.method_defined?(:request)
+    RestClient::Response.send(:include, RequestAccessor) unless RestClient::Response.method_defined?(:request)
```

Comment:
Yes, but it seems that `RestClient::Response` is declared before I extend the `AbstractResponse` so that my extension does not make it to the `Response`. Both of the 'classes' ere actually modules so there is no inheritance between them. I didn't find any better way to make this working. It could actually be possible not to extend `AbstractResponse` at all. But I didn't tested the consequences.